cmake/ExternalInclude.cmake

```diff
@@ -1,3 +1,14 @@
+#set the compile defdinition for current directory (i.e. externals)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  add_compile_options(/w )
+else()
+  add_compile_options(-w)
+endif()
+
+#create a sepefic options for the external build
+get_directory_property(EXTERNAL_FLAGS COMPILE_OPTIONS)
```

Comment:
Where is this directory property set ?

Reply:
`COMPILE_OPTIONS` is a cmake variable.

Reply:
line 3 or 5

Reply:
COMPILE_OPTIONS is not a cmake variable but a property name. but yes, it is set by `add_compile_option` and i missed it. but has it does nothing on my runs, i suspected a not set property